spring-cloud-kubernetes-dependencies/pom.xml

```diff
@@ -246,45 +246,6 @@
 				<scope>test</scope>
 			</dependency>
 
```

Comment:
The very first dependency that we have in `dependencyManagement` is :

```
<dependencyManagement>
		<dependencies>
			<dependency>
				<groupId>io.fabric8</groupId>
				<artifactId>kubernetes-client-bom</artifactId>
				<version>${kubernetes-fabric8-client.version}</version>
				<type>pom</type>
				<scope>import</scope>
			</dependency>
......
```

this is good, we do rely on the bom. But then, why is:


```
         <dependency>
				<groupId>io.fabric8</groupId>
				<artifactId>mockwebserver</artifactId>
				<version>${mockwebserver.version}</version>
				<scope>test</scope>
				<exclusions>
					<exclusion>
						<groupId>com.fasterxml.jackson.core</groupId>
						<artifactId>jackson-databind</artifactId>
					</exclusion>
					<exclusion>
						<groupId>com.fasterxml.jackson.core</groupId>
						<artifactId>jackson-core</artifactId>
					</exclusion>
					<exclusion>
						<artifactId>tools</artifactId>
						<groupId>com.sun</groupId>
					</exclusion>
				</exclusions>
			</dependency>
```

here? Bom already comes with:

```
<dependency>
   <groupId>io.fabric8</groupId>
   <artifactId>kubernetes-server-mock</artifactId>
   <version>5.9.0</version>
</dependency>
```

and Rohan in this blog post : https://itnext.io/mock-kubernetes-api-server-in-java-using-fabric8-kubernetes-mock-server-81a75cf6c47c, says (about `io.fabric8:kubernetes-server-mock`):

>  It’s a library based on Fabric8 Kubernetes Client and Fabric8 Mockwebserver(extension of OkHttp’s mockwebserver) which provides us smooth dsl for mocking operations...

So the question I had, why do we explicitly need `io.fabric8:mockwebserver` vs `io.fabric8:kubernetes-server-mock`? Let's figure it out:

```
bash-3.2$ read  -d ' ' x < <(grep -r "<artifactId>mockwebserver</artifactId>" .);echo "$x"
./spring-cloud-kubernetes-dependencies/pom.xml:                         <artifactId>mockwebserver</artifactId>
./spring-cloud-kubernetes-fabric8-config/pom.xml:                       <artifactId>mockwebserver</artifactId> 
```

In turns out that `spring-cloud-kubernetes-fabric8-config` requires it, but is it really needed? I removed it, ran `mvn clean test` - pass => safe to be deleted.